src/toil/test/jobStores/jobStoreTest.py

```diff
@@ -246,6 +246,9 @@ def test(self):
                 master.updateFile(fileOne, path)
                 with worker.readFileStream(fileOne) as f:
                     self.assertEquals(f.read(), 'two')
+            except shutil.Error as err:
```

Comment:
Do not like it. There isn't even an immediate shutil invocation anywhere in the try block. Let's discuss.